dotnet/samples/KernelSyntaxExamples/Example09_FunctionTypes.cs

```diff
@@ -95,109 +86,60 @@ public async Task<string> Type03Async()
     }
 
     [KernelFunction]
-    public void Type04(ContextVariables variables)
-    {
-        Console.WriteLine("Running function type 4");
-    }
-
-    [KernelFunction]
-    public string Type05(ContextVariables variables)
-    {
-        Console.WriteLine("Running function type 5");
-        return "";
-    }
-
-    [KernelFunction]
-    public async Task<string> Type06Async(Kernel kernel)
+    public async Task<string> Type04Async(Kernel kernel)
     {
-        var summary = await kernel.InvokeAsync(kernel.Plugins["SummarizePlugin"]["Summarize"], new ContextVariables("blah blah blah"));
-        Console.WriteLine($"Running function type 6 [{summary?.GetValue<string>()}]");
+        var summary = await kernel.InvokeAsync(kernel.Plugins["SummarizePlugin"]["Summarize"], new KernelFunctionArguments() { { "input", "blah blah blah" } });
+        Console.WriteLine($"Running function type 4 [{summary?.GetValue<string>()}]");
```

Comment:
Could this just be:
```suggestion
        Console.WriteLine($"Running function type 4 [{summary}]");
```
and rely on FunctionResult's ToString?